### PR TITLE
Store stars

### PR DESCRIPTION
**Pull request by esheldon**

### CODE REVIEW

File: pizza_cutter/des_pizza_cutter/_constants.py [hunk 1]
```diff
@@ -8,17 +8,13 @@
 # kernel extends outside bounds in resampling
 BMASK_RESAMPLE_BOUNDS = 2**26
 
-# this bit marks masked areas around gaia stars. Note this is different from
-# the STAR bit that is already set for masked areas around saturated stars
-
-BMASK_GAIA_STAR = 2**25
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_constants.py:14]: I'd argue that we want to keep this flag defined here. I know it is funny, but it makes sense to keep it with the rest of the flags.

File: pizza_cutter/des_pizza_cutter/_constants.py [hunk 2]
```diff
@@ -8,17 +8,13 @@
...
-
 # these are constants that are etched in stone for MEDS files
 MAGZP_REF = 30.0
 OBJECT_DATA_EXTNAME = 'object_data'
 IMAGE_INFO_EXTNAME = 'image_info'
 METADATA_EXTNAME = 'metadata'
 EPOCHS_INFO_EXTNAME = 'epochs_info'
+STARS_EXTNAME = 'stars'
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_constants.py:17]: Should we call this gaia stars?

File: pizza_cutter/des_pizza_cutter/_constants.py (hunk 1 above)
Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_constants.py:14]: OK

File: pizza_cutter/des_pizza_cutter/_constants.py (hunk 2 above)
Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_constants.py:17]: ok

**Review by beckermr — APPROVED** [review]

**Pull request merged**